test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/003_GENERIC_ERROR_in_case_HMI_respond_with_invalid_data.lua

```diff
@@ -0,0 +1,42 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description:
+-- In case:
+-- 1) App sends Get/Subscribe/UnsubscribeVehicleData(FuelRange:true) request
+-- 2) SDL transfers this request to HMI
+-- 3) HMI sends invalid response in `FuelRange` structure of Get/Subscribe/UnsubscribeVehicleData response
```

Comment:
Here should be checked only GetVD. SubscribeVD and UnsubscribeVD should not be checked with invalid values